### ISSUE TITLE
Guidance on engaging the community for contributions

### ISSUE THREAD

**Issue by NoureenS** [canada-ca/open-source-logiciel-libre#52]
For section: https://github.com/canada-ca/open-source-logiciel-libre/blob/master/en/guides/contributing-to-open-source-software.md#verify-contributing-process-and-policies

Is there a need to mention any GC requirements for business review of intended contribution to ensure IP protection? For example, guidance can say any bug fixes, don't require GC business review. However, for any substantial contributions that include new features, a chat with GC business leader/legal team may be required?

Add some additional guidance on how to contribute with impact. Questions to ask such as:
- What are the coding conventions of this community?
- Is there a roadmap that includes your direction?
- Has your proposal come up before?
- Is the project actively maintained?

For small changes including bug fixes --> simple pull requests
For large contributions --> engage community first to understand their interest and get any feedback before spending significant time and effort on the contribution
Communities may have different forms of engagement: creating an issue in GitHub, mailing list, Slack channel or some discussion channel

**Comment by gcharest** [comment]
Yes, very good point. Feature contribution Vs bug fix, documentation and
others have different involvment levels.

The process may need additional guidance for departments.
